qa/integration/services/service_container.rb

```diff
@@ -0,0 +1,66 @@
+require_relative "service"
+require "docker"
+
+# Represents a service running within a container.
+class ServiceContainer < Service
+
+  def initialize(name, settings)
+    super(name, settings)
+
+    @base_image_context = File.expand_path("../dockerized", __FILE__)
+
+    @image_context = File.join(@base_image_context, @name)
+
+    # Options to create the container.
+    @container_create_opts = {}
+
+    # Options to delete the container.
+    @container_delete_opts = {:force => true, :volumes => true}
```

Comment:
Indeed I was over-designing with that field. In the future, if ever needed, we can manage to add such as field back but keep force/clean always in.